include/arch/sparcv8leon3/sparcv8leon3.h

```diff
@@ -17,6 +17,15 @@
 #define _PHOENIX_ARCH_SPARCV8LEON3_H_
 
 
+#if defined(__CPU_GR716)
+#include "gr716/gr716.h"
```

Comment:
Will this include path be valid in userspace? It's <phoenix/arch/sparcv8leon3/gr716/*> there

Reply:
yeah, and this header is in `<phoenix/arch/sparcv8leon3/.>` so relative path will work (project builds fine)